pkg/highlight/parser.go

```diff
@@ -408,22 +412,44 @@ func parseRegion(group string, regionInfo map[interface{}]interface{}, prevRegio
 	r.group = groupNum
 	r.parent = prevRegion
 
-	r.start, err = regexp.Compile(regionInfo["start"].(string))
+	// start is mandatory
+	if _, ok := regionInfo["start"]; ok {
+		start := regionInfo["start"].(string)
```

Comment:
I think we could avoid doing the map lookup twice
```suggestion
	if start, ok := regionInfo["start"]; ok {
		start := start.(string)
```

Reply:
+1